### PR TITLE
{bio}[foss/2021a] BgeeDB v2.26.0

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@SIB-software:
`WARNING build failed (first 300 chars): Couldn't find file topGO_2.52.0.tar.gz anywhere, and downloading it didn't work either`

**Comment by smoretti** [comment]
@verdurin the topGO tarball link looks fine.
Could you restart the bot?

File: easybuild/easyconfigs/b/BgeeDB/BgeeDB-2.26.0-foss-2021a-R-4.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+dependencies = [
+    ('R-bundle-Bioconductor', '3.13', versionsuffix),
+]
+
+exts_default_options = {
+    'source_urls': [
+        'https://bioconductor.org/packages/%(local_bioc_version)s/bioc/src/contrib/',
+        'https://bioconductor.org/packages/%(local_bioc_version)s/bioc/src/contrib/Archive/%(name)s',
+        'https://bioconductor.org/packages/%(local_bioc_version)s/data/annotation/src/contrib/',
+        'https://bioconductor.org/packages/%(local_bioc_version)s/data/experiment/src/contrib/',
```
Comment (branfosj) [inline easybuild/easyconfigs/b/BgeeDB/BgeeDB-2.26.0-foss-2021a-R-4.1.0.eb:32]:
```suggestion
        'https://bioconductor.org/packages/%s/bioc/src/contrib/' % local_bioc_version,
        'https://bioconductor.org/packages/%s/bioc/src/contrib/Archive/%%(name)s' % local_bioc_version,
        'https://bioconductor.org/packages/%s/data/annotation/src/contrib/' % local_bioc_version,
        'https://bioconductor.org/packages/%s/data/experiment/src/contrib/' % local_bioc_version,
```

Local variables do not get replaced, so have to be used as per the suggested replacement instead.

Comment (branfosj) [reply easybuild/easyconfigs/b/BgeeDB/BgeeDB-2.26.0-foss-2021a-R-4.1.0.eb:32]: So in the test report we see it trying to download with the `%(local_bioc_version)s` inthe URL:

```
== 2023-09-08 09:55:32,644 filetools.py:832 WARNING URL https://bioconductor.org/packages/%(local_bioc_version)s/bioc/src/contrib/topGO_2.52.0.tar.gz was not found (HTTP response code 400), not trying again
```

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/273f195f88a8a845d728f9dd748d84a8 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @SIB-software!

**Pull request merged**